pkgs/development/libraries/gstreamer/core/default.nix

```diff
@@ -21,8 +21,9 @@
 , buildPackages
 , withIntrospection ? lib.meta.availableOn stdenv.hostPlatform gobject-introspection && stdenv.hostPlatform.emulatorAvailable buildPackages
 , libunwind
-# darwin.libunwind doesn't have pkg-config definitions so meson doesn't detect it.
-, withLibunwind ? !stdenv.isDarwin && lib.meta.availableOn stdenv.hostPlatform libunwind
+, withLibunwind ?
+  lib.meta.availableOn stdenv.hostPlatform libunwind &&
+    lib.elem "libunwind" libunwind.meta.pkgConfigModules or []
```

Comment:
Nit: I find `f x.a or b` to be a really confusing parse, and would prefer parentheses around the relevant subexpression here. (Non‐blocking, will merge soon regardless of any response if you don’t beat me to it.)

Reply:
I also find it confusing, but unless we can enforce parentheses treewide, I think it's better to teach it through exposure, rather than making it rare and therefore even more surprising.

Reply:
Opened https://github.com/NixOS/nixfmt/issues/251 and merging :)